### PR TITLE
Update schema.task_execution_service.d55bf88.openapi.modified.yaml

### PR DESCRIPTION
**Pull request by vaniisgh**
add tesResources to /service-info/tasks-info endpoint

### CODE REVIEW

**Review by uniqueg — CHANGES_REQUESTED** [review]
There's a couple of things:
- Don't forget to  replace `execution_time` with `costs`.
- We will **not** need a property `id` in the response; no identifier will be generated for a request to `GET /service-info/task-info`.
- Think about the types of the properties; just putting `string` is *not* a good idea as it would allow any TES implementation to just report back costs and time in any way they like, making it virtually impossible for us to interpret these values; check online how this can be handled better for OpenAPI/Swagger specs; if you don't find anything, you may need to write a model to represent costs (including currency!) and time in safer ways (i.e., with e.g. integers or floats for values and controlled vocabulary, if necessary).
- Please include meaningful descriptions for your properties. We may know now what they are supposed to be, but other people joining in later most likely will not know. This goes for documentation in general. It may seem easier to do this later, but then these things tend to be forgotten.
- It appears that most of the changes in this pull request are deletions of whitespace (8 deletions and insertions). While this may not affect the validity of the specs, it is a bit annoying because it makes it harder to review changes. This may not seem like a big problem now, but if we end up deciding to file a pull request to the original TES specs repo, it could get very confusing, especially if it results in conflicts (because the official specs might move on in the meantime). So please add back the missing whitespaces before we pull in the changes. Also, if you didn't do these whitespace deletions manually, please check the settings of your editor.

Thanks!

**Comment by vaniisgh** [comment]
Made a few modifications, I'm still not completely sure they'll cover all the cases ... would be great if you could review.

File: specs/schema.task_execution_service.d55bf88.openapi.modified.yaml [hunk 1]
```diff
@@ -600,15 +616,20 @@ definitions:
...
-        description: ''
-      execution_time:
-        type: string
-        description: ''
+      cost:
+        $ref: '#/definitions/tesCost'
       queuing_time:
+        type: number
+        format: int64
+        description: 'Time required for the running of the'
```
Comment (uniqueg) [inline specs/schema.task_execution_service.d55bf88.openapi.modified.yaml]: Time in which unit, seconds?

Comment (vaniisgh) [reply specs/schema.task_execution_service.d55bf88.openapi.modified.yaml]: queuing_time_unit parameter should cover that .. I didn't want to add a new model .. not sure why it doesn't show up here ... will just update

**Comment by uniqueg** [comment]
I've changed a few things, most importantly I've added the resource requirements _as input parameters_ to the endpoint. Without that, we couldn't get a meaningful estimate. Other changes are just the descriptions.

Can you test whether the modified specs are valid (connexion and bravado do not complain)? Once the tests are okay, I think we can merge.

**Comment by vaniisgh** [comment]
I see I completely missed that point. 
Thank you! 

The specs are valid as well.

**Comment by uniqueg** [comment]
Great, I'll pull it

**Pull request merged**